run_trial.bash

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+#TODO: RE-ENABLE THIS. (dhood): it was disabled because when the server container shuts down, it appears to return non-zero.
+# set -e
+
+TEAM_NAME=$1
+TRIAL_NAME=$2
+
+#TODO: generate unique container names based on input arguments
+CONTAINER_NAME=ariac-server-system
+
+HOST_LOG_DIR=`pwd`/logs/${TEAM_NAME}/${TRIAL_NAME}
+echo "Creating directory: ${HOST_LOG_DIR}"
+mkdir -p ${HOST_LOG_DIR}
+
+# TODO: don't rely on script being run in the root directory
+# TODO: error checking for case when files can't be found
+TEAM_CONFIG_DIR=`pwd`/${TEAM_NAME}
+echo "Using team config: ${TEAM_CONFIG_DIR}/team_config.yaml"
+COMP_CONFIG_DIR=`pwd`/comp_configs
+echo "Using comp config: ${COMP_CONFIG_DIR}/${TRIAL_NAME}.yaml"
+
+LOG_DIR=/ariac/logs
+
+# Create the network for the containers to talk to each other.
+./ariac-competitor/ariac_network.bash
+
+# Start the competitors container and let it run in the background.
+#TODO: parameterize the container name
+./ariac-competitor/run_competitor_container.bash &
+
+# Start the competition server. When the trial ends, the container will be killed.
+# The trial may end because of time-out, because of completion, or because the user called the
+# /ariac/end_competition service.
+./ariac-server/run_container.bash ${CONTAINER_NAME} ariac-server \
+  "-v ${TEAM_CONFIG_DIR}:/team_config \
+  -v ${COMP_CONFIG_DIR}:/ariac/comp_configs \
+  -v ${HOST_LOG_DIR}:${LOG_DIR} \
+  -e ARIAC_EXIT_ON_COMPLETION=1" \
+  "/run_ariac_task.sh /ariac/comp_configs/${TRIAL_NAME}.yaml /team_config/team_config.yaml ${LOG_DIR}"
+
+docker kill ariac-competitor-system
+
+# Make the log playable
+#TODO: figure out why the ownership of the state log is root and resolve.
+echo "Changing ownership of gazebo state log"
+sudo chown $USER ${HOST_LOG_DIR}/gazebo/state.log
```

Comment:
@j-rivero, I noticed that the gazebo state log has `root` ownership. The log is generated in the container, and then copied into `/ariac/logs` on the container [in this script](https://github.com/osrf/ariac-docker/pull/6/files#diff-861a436c3d2af41ee884b625a77cd85aR47), which is [mounted to `logs/{team_name}/{trial_name}` on the host](https://github.com/osrf/ariac-docker/pull/6/files#diff-c2e963c2b9be2d58754e5afa23675caeR38). Do you have any idea what might cause this?

This workaround is necessary to make the log playable on the host. If we can avoid it, the script won't need `sudo` I believe.